sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-group-by.sql

```diff
@@ -0,0 +1,117 @@
+-- A test suite for GROUP BY in parent side, subquery, and both predicate subquery
+-- It includes correlated cases.
+
+-- tables and data types
+
+CREATE DATABASE indb;
```

Comment:
For each test suite, (e.g., in-group-by.sql), we create a dedicated sesssion. See [the code](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala#L144-L147). 

Why not creating temporary views? These views will be automatically removed after the session ends.